04 Strategy Library/207 Value Effect within Countries/02 Method.html

```diff
@@ -0,0 +1,87 @@
+<p>
+  This new metric is called the cyclically adjusted price-to-earnings ratio (CAPE). The basic idea is comparing the stock prices with earnings smoothed across multiple years.
+  Using backward-looking earnings allows us to smooth out the economic cycle as well as the price fluctuations.
+  By definition, CAPE is the price divided by the average of ten years of earnings (moving average), adjusted for inflation.
```

Comment:
- The algorithm takes the previous 10 years of earnings to smooth out the economic cycle as well as the price fluctuations.
 - Drop "By definition,"